src/main/java/roomescape/reservation/dto/request/ThemeRequest.java

```diff
@@ -1,10 +1,17 @@
-package roomescape.dto.request;
+package roomescape.reservation.dto.request;
 
 import jakarta.validation.constraints.NotBlank;
+import roomescape.reservation.domain.Theme;
 
 public record ThemeRequest(
         @NotBlank(message = "name 값이 없습니다.") String name,
         @NotBlank(message = "description 값이 없습니다.") String description,
         @NotBlank(message = "thumbnail 값이 없습니다.") String thumbnail
 ) {
+    public Theme toTheme() {
+        return Theme.builder()
+                .name(name)
+                .description(description)
+                .thumbnail(thumbnail).build();
```

Comment:
팀마다 컨벤션은 다르겠지만, 점이 여러 개인 경우 한 줄에 하나만 찍는 것이 일반적입니다 😄 

```suggestion
                .thumbnail(thumbnail)
                .build();
```

Reply:
넵 수정했습니다!